packages/jsio.js

```diff
@@ -370,6 +363,17 @@
       this.eval = function(code, path) {};
       this.fetch = function(path) { return contentsOfPath; };
       this.log = function(args...) {};
+
+      this.getNamespace = function(key) { return CONFIG.shortName + ':' + key };
```

Comment:
maybe getNamespace should live outside of jsio since it seems to be devkit-specific